ocs_ci/ocs/node.py

```diff
@@ -2350,3 +2350,26 @@ def wait_for_new_worker_node_ipi(machineset, old_wnodes, timeout=900):
     wait_for_nodes_status([new_wnode.name])
     log.info(f"The new worker node {new_wnode.name} is in a Ready state!")
     return new_wnode
+
+
+def check_for_zombie_process_on_node(node_name=None):
+    """
+    Check if there are any zombie process on the nodes
+
+    Args:
+        node_name (list): Node names list to check for zombie process
+
+    Raise:
+        ZombieProcessFoundException: In case zombie process are found on the node
+
+    """
+    node_obj_list = get_node_objs(node_name) if node_name else get_node_objs()
+    for node_obj in node_obj_list:
+        debug_cmd = f'debug nodes/{node_obj.name} -- chroot /host /bin/bash -c "top -b1 -n1 | grep Z"'
+        out = node_obj.ocp.exec_oc_cmd(command=debug_cmd, out_yaml_format=False)
+        log.info(f"Zombie process check command output: {out}")
+        if not out:
+            log.info(f"No Zombie process found on the node: {node_obj.name}")
+        else:
+            log.error(f"Zombie process found on node: {node_obj.name}")
```

Comment:
Please add the process name for later analysis

Reply:
Done. Logging zombie  pid and ppid details